index.html

```diff
@@ -1,21 +1,289 @@
+            <span class="chat-icon"></span>
+          </section>
+        </section>
+        <section class="products">
+          <div class="feature-and-image">
+            <div class="feature image-products">
+              <div class="feature-title-btn">
+                <h2>CACTI</h2>
+                <div class="border-bottom-h2"></div>
+                <button class="btn">Shop Collection</button>
+              </div>
+              <div class="feature-img" title="">
+                <img
+                  src="../responsive-grid-project/img/first-image.jpg"
+                  alt=""
+                />
+              </div>
+            </div>
+            <div class="feature image-products">
+              <div class="feature-title-btn">
+                <h2>PLANTS</h2>
+                <div class="border-bottom-h2"></div>
+                <button class="btn">Shop Collection</button>
+              </div>
+              <div class="feature-img" title="">
+                <img
+                  src="../responsive-grid-project/img/second-image.jpg"
+                  alt=""
+                />
+              </div>
+            </div>
+            <div class="feature image-products">
+              <div class="feature-title-btn">
+                <h2>SUCCULENTS</h2>
+                <div class="border-bottom-h2"></div>
+                <button class="btn">Shop Collection</button>
+              </div>
+              <div class="feature-img" title="">
+                <img
+                  src="../responsive-grid-project/img/third-image.jpg"
+                  alt=""
+                />
+              </div>
+            </div>
+          </div>
+          <div class="feature">
+            <div id="about" class="about">
+              <div class="about-us-text">
+
+                <h3 class="about-us">ABOUT US</h3>
+                <div class="border-bottom-h2"></div>
+                <p>
+                  I'm a paragraph. Click here to add your own text and edit me.
+                  It’s easy. Just click “Edit Text” or double click me to add your
+                  own content and make changes to the font. Feel free to drag and
+                  drop me anywhere you like on your page. I’m a great place for
+                  you to tell a story and let your users know a little more about
+                  you.
+                  <br /><br />
+                  This is a great space to write long text about your company and
+                  your services. You can use this space to go into a little more
+                  detail about your company. Talk about your team and what
+                  services you provide.
+                </p>
+              </div>
+              <div class="img-about">
+                <img
+                  class="img-area-about"
+                  src="/home/marina/Desktop/CodeYouFuture/responsive-grid-project/img/mobile-area-img.jpg"
```

Comment:
This is a path to an image on **your** computer. No one else has access to `/home/marina/Desktop` therefore no image is showing up for me. 
Replace it with the **relative** path to the image inside the project folder. It should match the same folder structure on Github, and therefore should be able to access the image. Example: `./img/mobile-area-img.jpg`